### ISSUE TITLE
Strange OperatingSystem Name/Version for Windows

### ISSUE THREAD

**Issue by DaimonPl** [nielsbasjes/yauaa#102]
Hi,

I wanted to extract operating system parameters using `OPERATING_SYSTEM_NAME` and `OPERATING_SYSTEM_VERSION` so I can group data by OS name and then by version.

However it works very strange for windows:
```
Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/63.0.3239.132 Safari/537.36
OPERATING_SYSTEM_NAME = "Windows NT"
OPERATING_SYSTEM_VERSION = "Windows 10.0"
```

Why `OPERATING_SYSTEM_VERSION` is not just `10.0` ? It's confusing because by combining name and version I'm getting `Windows NT Windows 10.0` instead of `Windows NT 10.0`. I'm aware that in that case `Name Version` field could be used but it's not generic approach when comparing with other OS/Versions

**Comment by nielsbasjes** [comment]
I have been having problems getting this "right" and to make the names and versions "workable" I came up with this scheme. As you mentioned exactly to fix this concatenation problem you describe I have already added the OperatingSystemNameVersion field which is the correct concatenation.

I have to think about this.

**Comment by nielsbasjes** [comment]
I'm closing this one as "Fixed"

**Comment by nielsbasjes** [comment]
Yesterday I released version 5.1 which includes this (and much more). 
Thanks again for reporting.
